_pages/docs/merging-options.md

```diff
@@ -62,27 +74,15 @@ retrieved as either XML or JSON.
       <thesaurus>/mdm/config/thesauri/first-name-synonyms.xml</thesaurus>
       <length weight="8" />
     </merge>
-    <merge property-name="address" algorithm-ref="standard" max-values="1">
-      <source-weights>
-        <source name="CRM" weight="10"></source>
-      </source-weights>
-    </merge>
-    <merge property-name="dob" algorithm-ref="standard" max-values="1">
-      <source-weights>
-        <source name="Oracle" weight="10"></source>
-      </source-weights>
-    </merge>
-    <merge property-name="caseStartDate" algorithm-ref="standard" max-values="1">
-      <source-weights>
-        <source name="CRM" weight="10"></source>
-      </source-weights>
-    </merge>
-    <merge property-name="incidentDate" algorithm-ref="standard" max-values="1">
-      <length weight="10"/>
-    </merge>
-    <merge property-name="sex" algorithm-ref="standard" max-values="1">
-      <length weight="10"/>
-    </merge>
+    <merge property-name="address" strategy="crm-source-weight" max-values="1"></merge>
+    <merge property-name="dob" strategy="oracle-source-weight" max-values="1"></merge>
```

Comment:
I think at least one of these examples should continue to use `@algorithm-ref`. How about dropping the `oracle-source-weight` (since it's only used once) and having that one set up the properties the previous way (which I presume still works)?